Makefile

```diff
@@ -48,38 +56,41 @@ clean-dockerdemo:
 ssh:
 	@vagrant ssh netplugin-node1 || echo 'Please run "make demo"'
 
-unit-test: build
-	CONTIV_HOST_GOPATH=$(GOPATH) CONTIV_HOST_GOBIN=$(HOST_GOBIN) \
-					   CONTIV_HOST_GOROOT=$(HOST_GOROOT) ./scripts/unittests -vagrant
+unit-test: stop clean build
```

Comment:
yeah, my concern is mostly around increased test times and you bring up a good point around consistent toolchain. 

We can keep this as is for now until we run into slow build/test times.